app/src/main/java/com/kamron/pogoiv/OcrHelper.java

```diff
@@ -92,8 +94,8 @@ private boolean isCandyWordFirst() {
      * @param simpleBG     Whether the bitmap has a simple background
      * @return Bitmap with replaced colors
      */
-    private Bitmap replaceColors(Bitmap srcBitmap, int keepCr, int keepCg, int keepCb, int replaceColor, int distance,
-                                 boolean simpleBG) {
+    private Bitmap replaceColors(Bitmap srcBitmap, Bitmap dstBitmap, int keepCr, int keepCg, int keepCb,
```

Comment:
I think a "create copy" or some sort of boolean would be a much clearer API. You also won't have to depend or assume if the input bitmap is the same side.